src/test/java/com/example/universe/simulator/entityservice/integration/GalaxyIntegrationTest.java

```diff
@@ -83,6 +91,15 @@ void test() throws Exception {
             .contentType(MediaType.APPLICATION_JSON)
             .content(objectMapper.writeValueAsString(dto))
         );
+
+        //check if resultDto got updated in cache
+        GalaxyDto fromCacheUpdated = modelMapper.map(
+                objectMapper.convertValue(redisTemplate.opsForValue().getAndDelete("Galaxy::" + addedDto1.getId()), Galaxy.class),
+                GalaxyDto.class);
+        assertThat(fromCacheUpdated.getName()).isEqualTo(dto.getName());
+        assertThat(fromCacheUpdated.getVersion()).isEqualTo(dto.getVersion() + 1);
```

Comment:
no need to increment if checked with the result dto

Reply:
we have reference of old version at that time, how should we compare it then?

Reply:
from response of `performRequest(put("/galaxy/update")`